ci_framework/roles/devscripts/tasks/03_install.yml

```diff
@@ -51,12 +51,23 @@
     dest: >-
       {{ cifmw_devscripts_repo_dir }}/config_{{ cifmw_devscripts_user }}.sh
 
-- name: Copy the user pull secret
+- name: Ensure pull-secret is available to the env
   tags:
     - bootstrap
   ansible.builtin.copy:
     dest: "{{ cifmw_devscripts_repo_dir }}/pull_secret.json"
-    content: "{{ cifmw_devscripts_pull_secret }}"
+    content: >-
```

Comment:
interesting... apparently the way "omit" is set here fails for some reason, seeing the molecule output :(. Might need to be two tasks with an if after all. Ugly, dumb, but heh... guess that's how things are to be.